code/modules/vehicles/mecha/combat/honker.dm

```diff
@@ -5,140 +5,24 @@
-					"}
-
-/obj/vehicle/sealed/mecha/combat/honker/get_ssticker_function()
-	. = {"
-		function SSticker() {
-		    setInterval(function(){
-		        window.location='byond://?src=[REF(src)]&update_content=1';
-		        document.body.style.color = get_rand_color_string();
-		      document.body.style.background = get_rand_color_string();
-		    }, 1000);
-		}
-
-		function get_rand_color_string() {
-		    var color = new Array;
-		    for(var i=0;i<3;i++){
-		        color.push(Math.floor(Math.random()*255));
-		    }
-		    return "rgb("+color.toString()+")";
-		}
-	"}
-
-/obj/vehicle/sealed/mecha/combat/honker/get_commands()
-	var/output = {"<div class='wr'>
-						<div class='header'>Sounds of HONK:</div>
-						<div class='links'>
-						<a href='?src=[REF(src)];play_sound=sadtrombone'>Sad Trombone</a>
-						<a href='?src=[REF(src)];play_sound=bikehorn'>Bike Horn</a>
-						<a href='?src=[REF(src)];play_sound=airhorn2'>Air Horn</a>
-						<a href='?src=[REF(src)];play_sound=carhorn'>Car Horn</a>
-						<a href='?src=[REF(src)];play_sound=party_horn'>Party Horn</a>
-						<a href='?src=[REF(src)];play_sound=reee'>Reee</a>
-						<a href='?src=[REF(src)];play_sound=weeoo1'>Siren</a>
-						<a href='?src=[REF(src)];play_sound=hiss1'>Hissing Creature</a>
-						<a href='?src=[REF(src)];play_sound=armbomb'>Armed Grenade</a>
-						<a href='?src=[REF(src)];play_sound=saberon'>Energy Sword</a>
-						<a href='?src=[REF(src)];play_sound=airlock_alien_prying'>Airlock Prying</a>
-						<a href='?src=[REF(src)];play_sound=lightningbolt'>Lightning Bolt</a>
-						<a href='?src=[REF(src)];play_sound=explosionfar'>Distant Explosion</a>
-						</div>
-						</div>
-						"}
-	output += ..()
-	return output
-
-
-/obj/vehicle/sealed/mecha/combat/honker/get_equipment_list()
-	if(!LAZYLEN(equipment))
-		return
-	var/output = "<b>Honk-ON-Systems:</b><div style=\"margin-left: 15px;\">"
-	for(var/obj/item/mecha_parts/mecha_equipment/MT in equipment)
-		output += "<div id='[REF(MT)]'>[MT.get_equip_info()]</div>"
-	output += "</div>"
-	return output
-
-/obj/vehicle/sealed/mecha/combat/honker/get_equipment_menu() //outputs mecha html equipment menu
-	. = {"
-	<div class='wr'>
-	<div class='header'>Honk-ON-Systems</div>
-	<div class='links'>"}
-	for(var/e in equipment)
-		var/obj/item/mecha_parts/mecha_equipment/equipment = e
-		. += "[equipment.name] [equipment.detachable ? "<a href='?src=[REF(equipment)];detach=1'>Detach</a><br>" : "\[Non-removable\]<br>"]"
-	. += {"<b>Available HONK slots:</b> [max_equip-LAZYLEN(equipment)]
-	</div>
-	</div>"}
-
 /obj/vehicle/sealed/mecha/combat/honker/play_stepsound()
 	if(squeak)
 		playsound(src, "clownstep", 70, 1)
 	squeak = !squeak
 
-/obj/vehicle/sealed/mecha/combat/honker/Topic(href, href_list)
-	..()
-	if (href_list["play_sound"])
-		switch(href_list["play_sound"])
```

Comment:
this removal needs to be in the changelog

Reply:
Done